### ISSUE TITLE
TortoiseSVN "ignore-on-commit" feature

### ISSUE THREAD

**Issue by JohnstonCode** [JohnstonCode/svn-scm#99]
This requires changelist implementation to work.

[Docs here](https://tortoisesvn.net/docs/release/TortoiseSVN_en/tsvn-dug-commit.html)

**Comment by Guema** [comment]
Do you plan to add a third category ?
like : 
"CHANGES"
"NOT TRACKED"
"IGNORE ON COMMIT"

I would make good use of it, personally, i think this is a good feature.
Could be interesting to make categories collapsible, also, but i think it's not a priority.

**Comment by JohnstonCode** [comment]
Just waiting on changelist implementation for this to work.

Not sure that SCM groups can be collapsable
